android/src/main/java/com/reactnativecommunity/webview/RNCWebViewManager.java

```diff
@@ -1622,6 +1622,11 @@ protected void onSizeChanged(int w, int h, int ow, int oh) {
     @Override
     public void setWebViewClient(WebViewClient client) {
       super.setWebViewClient(client);
+
+      if (client == null) {
+        mRNCWebViewClient = null;
+      }
+
       if (client instanceof RNCWebViewClient) {
```

Comment:
nit: why not combine these via `else if`? (if `client` is `null`, then surely we already know that it can't be an instance of `RNCWebViewClient`..)